content/applications/hr/recruitment/new_job.rst

```diff
@@ -44,40 +50,43 @@ options, and then click :guilabel:`Configuration` to edit the details.
 
-Enter the job description in the :guilabel:`Job Summary` tab. This information is what is visible to
-potential applicants when searching for available jobs.
-
-Recruitment
-~~~~~~~~~~~
-
 All the basic information about the job position is listed under the :guilabel:`Recruitment` tab.
-None of the fields are required, but it is a good idea to provide at least a few details, such as
-where the job is located.
+
+None of the fields are required, but it is important to configure and populate the
+:guilabel:`Department`, :guilabel:`Location`, :guilabel:`Employment Type`, and :guilabel:`Job
+Summary` fields, as they are all visible to prospective applicants on the website.
 
 The fields can be filled out as follows:
 
-- :guilabel:`Department`: select the relevant department for the job position.
-- :guilabel:`Job Location`: select the physical address for the job.
-- :guilabel:`Employment Type`: select what type of position the job is, such as
-  :guilabel:`Full-Time`, :guilabel:`Part-Time`, etc.
-- :guilabel:`Company`: select the company the job is for.
+- :guilabel:`Department`: select the relevant department for the job position. This is visible on
+  the website.
+- :guilabel:`Job Location`: select the physical address for the job. If the job position is remote,
+  leave this field blank. This is visible on the website.
+- :guilabel:`Email Alias`: enter an email address to which applicants can send a resumé. Once
+  emailed, Odoo automatically creates an application for them.
+- :guilabel:`Employment Type`: select what type of position the job is, using the drop-down menu.
+  The default options are :guilabel:`Permanent`, :guilabel:`Temporary`, :guilabel:`Seasonal`,
+  :guilabel:`Interim`, :guilabel:`Full-Time`, and :guilabel:`Part-Time`. This is visible on the
+  website.
+- :guilabel:`Company`: select the company the job is for. This field only appears if using a
+  multi-company database.
 - :guilabel:`Target`: enter the number of employees to be hired for this position.
 - :guilabel:`Is Published`: activate this option to publish the job online.
-- :guilabel:`Website`: select the website where the job will be published.
-- :guilabel:`Recruiter`: select the person who will be doing the recruiting for this role.
-- :guilabel:`Interviewers`: select who should perform the interview(s). Multiple people can be
+- :guilabel:`Website`: select the website the job is published on.
+- :guilabel:`Recruiter`: select the person responsible for recruiting this role.
+- :guilabel:`Interviewers`: select who should perform the interviews. Multiple people can be
   selected.
-- :guilabel:`Interview Form`: select an :ref:`Interview form <interview>` that applicants will fill
-  out prior to their interview.
-- :guilabel:`Contract Template`: select a contract template that will be used when offering the job
-  to a candidate.
+- :guilabel:`Interview Form`: select an :ref:`Interview form <interview>` that applicants fill out
+  prior to their interview.
```

Comment:
Please update the custom anchor to include the app/module name :)
```suggestion
- :guilabel:`Interview Form`: select an :ref:`Interview form <recruitment/interview>` that
  applicants fill out prior to their interview.
```

Reply:
I forgot about adding that first part! The funny thing is, it worked in the 'make fast' when I test it. I assumed if it did- then it was ok. Whoops =)